test/test-0018.sh

```diff
@@ -5,34 +5,38 @@
 cleanup 18
 
 # ------------------------------- Test 18 ------------------------------------
-preptest test.log 18 1
+needle="$(uuidgen -r)"
```

Comment:
How are you confirming that ```journalctl``` is unconditionally used?

The test passes just fine on my system where ```journalctl``` does not exist.  Not that I understand what the test actually checks but we need to fix it to exit with 77 if its unconditional requirements are not met.  As it works now, either the ```journalctl``` availability is optional, or the test is badly broken.